inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+. There, see the upper section.). If they do not answer your question,
+don‘t hesitate to contact our team: ____ We are glad to help you.
+** Whenever you find a blue questionmark-sign in INCEpTION, you may
+click on it to find further explanation in the main documentation.
+** For more details, see the installation guide
+** For more installation, see the admin guide. You find it on our
+website, among the tabs on top: → _Documentation_,
+https://inception-project.github.io/releases/0.10.0/docs/admin-guide.html[h]https://inception-project.github.io/releases/0.10.0/docs/admin-guide.html[ttps://inception-project.github.io/releases/0.10.0/docs/admin-guide.html]
 
-Alternatively, you can start it on the command line using
+Installing and starting INCEpTION
 
-[subs="+attributes"]
-----
-$ java -jar inception-app-standalone-{revnumber}.jar
-----
+* Installing Java
+** In order to make INCEpTION run, you will need to install Java (in
+case you do not have it yet). Here
+https://java.com/en/download/help/download_options.xml#linux you find
+instructions on how to install Java on Windows, Mac, Linux and Solaris.
 
-The splash screen and dialog will not appear in this case and you have to manually point your browser at `http://localhost:8080`.
+* Download and start INCEpTION
+** Now, download the .jar-file from our website
+https://inception-project.github.io/ . You will find the
+„Download“-Button top right. Wait a moment until it is fully downloaded.
+That is, until the name of the downlauded folder ends on „.jar“, not on
+„.jar.part“ anymore.
++
+_Note_: We recommend working on the latest version always. So check the
+website once in a while and download the new .jar-file when a later
+version is available.
+** Now, simply ***double-click ***on the downloaded .jar-file. After a
+moment, a splash screen will display while the application is
+initializing. Once the initialization is complete, a dialog appears.
+Here, you can open the application in your default browser or shut it
+down again.
+** If you are used to the command line, you may enter this command there
+instead:
++
+$ java -jar inception-app-standalone-0.10.0.jar
++
+In this case, no splash screen will start. Just enter
+http://localhost:8080/[_http://localhost:8080_] in you browser.
+** Use „admin“ as username and password to login, since a default user
+with these properties is created the first time you start the
+application.
 
-The first time you start the application, a default user with the name `admin` and the password
-`admin` is created. Use this username and password to log in to the application after opening it
-in your browser.
+* Download an example project for your first steps
+** Also on our website, clicking the tab „Example Projects“ on the very
+top leads you to the following page:
+https://inception-project.github.io/example-projects/ . To follow this
+Guide, please download the _Interactive Concept Linking_ project.
```

Comment:
Explain what this project does, what concept linking is etc

Reply:
Project: Fine like this?
Concept linking: I would not explain the task of annotation itself and anythis that has o do wth that but only our tool. When you buy a car, the instructions will not explain you how to drive (= what we learn in driving class) but how to drive _this car_. When I look for an annotation tool, I know the concepts I am going to annotate.
However, I explained it quickly, are you fine with that?

Reply:
This is the getting started guide for people who want to know what INCEpTION is, so I would assume nothing about the user. Its up to you, but I want this guide to be as inclusive as possible to reel everyone in.

Reply:
Yes, me too. On the one hand, I would love to explain the parts which dig into the annotation task itself, too, which I really struggled with when I was new, so our users will have a better experience. On the other hand, we have 10 (!!) pages already and people just don't like to read much when they want to get started with something. I blieve, we will loose users on this...
Shouldn't we assume, people know how to drive when they buy a car / know how to annotate when they look for an annotation tool?

I feel, we have different ideas for whom the guide shall be (what they know before), what situation they are in when they read it and what they should know afterwards. We should get on the same page with that.

Reply:
same like before: we changed tha approach and structure. should be good now.